healpy/pixelfunc.py

```diff
@@ -733,7 +733,7 @@ def ring2nest(nside, ipix):
     >>> hp.ring2nest([1, 2, 4, 8], 11)
     array([ 11,  13,  61, 253])
     """
-    check_nside(nside)
+    check_nside(nside, nest=False)
```

Comment:
This should be nest=True, I think, otherwise you cannot convert to nested scheme.